lib/ansible/modules/cloud/amazon/ecs_ecr.py

```diff
@@ -263,6 +351,46 @@ def run(ecr, params, verbosity):
                 result['created'] = True
             result['repository'] = repo
 
+            if delete_lifecycle_policy:
+                original_lifecycle_policy = \
+                    ecr.get_lifecycle_policy(registry_id, name)
+
+                if verbosity >= 2:
+                    result['lifecycle_policy'] = None
+
+                if verbosity >= 3:
+                    result['original_lifecycle_policy'] = original_lifecycle_policy
+
+                if original_lifecycle_policy:
+                    ecr.delete_lifecycle_policy(registry_id, name)
+                    result['changed'] = True
+
+            elif lifecycle_policy_text is not None:
+                try:
+                    lifecycle_policy = sort_json_policy_dict(lifecycle_policy)
+                    if verbosity >= 2:
+                        result['lifecycle_policy'] = lifecycle_policy
+                    original_lifecycle_policy = ecr.get_lifecycle_policy(
+                        registry_id, name)
+
+                    if original_lifecycle_policy:
+                        original_lifecycle_policy = sort_json_policy_dict(
+                            original_lifecycle_policy)
+
+                    if verbosity >= 3:
```

Comment:
Please don't change results based on verbosity, this is *very* unexpected.

Reply:
What's the recommended way to increase details with increased verbosity levels?

Reply:
Just return the same stuff for all requests. That's what I expect. 

This and ecs_ecr are the only modules in the entire codebase that return different results based on verbosity. It is not a pattern I would like to see expanded.

Reply:
Well, there's lxd_container, lxd_profile, nxos_gir, nxos_gir_profile_management. So I'm not completely alone 😁

Anyways, I take your point. Thinking about it more, what I *really* want here would actually be output when `--diff` is given. But that would be the topic of a different PR.

I'll get this cleaned up. Thanks!

Reply:
Fixed.